backend.py

```diff
@@ -661,7 +661,9 @@ def get_dealt_cards(self, robot):
             self.present_deck.extend(self.past_deck)
             shuffle(self.present_deck)
         dealt_cards = self.present_deck[-dealt_cards_count:]
+        # Delete cards from present deck and add them to deck with used ones.
         del self.present_deck[-dealt_cards_count:]
+        self.add_to_past_deck(dealt_cards)
```

Comment:
Mám pocit, že to teď nedělá ale to stejné ne?  Můžeš mi to slovně popsat tuhle část? 
Předtím to do past_deck přidávalo pouze karty, které si hráč nevybral, na začátku tedy 4 ks. 
Takto mi přijde že se do past_deck přesune všech 9 dealt_cards. Nebo chápu špatně co se dělo předtím?

Reply:
Předtím to bylo špatně, ty karty se jen smazaly, ale vsechny dealt cards maji jit do toho balíčku, jak jento tady ze?

Reply:
Teď se všechny rozdané karty rovnou přidají do balíčku odhozených karet. Ale to pořád neřeší všechny problémy :(. Blokované karty můžou být na ruce hráče dlouho, a zároveň se už může použit stejná karta z odhazovacího balíku. Takže to není finální řešení.

Reply:
Mě spíš zajimalo jak je to v té hře právě, takže by se měly přesunout ty dealt card kromě případných blokovaných. To se promysli :)

Reply:
Takže schvaluji a až zjistíme jak pracovat s těmi blokovanými kartami tak se to udělá dál.

Reply:
Takže schvaluji a až zjistíme jak pracovat s těmi blokovanými kartami tak se to udělá dál.